### ISSUE TITLE
CAW Implement 'remaining characters'

### ISSUE THREAD

**Issue by janeswalk** [jkoudys/janeswalk-web#110]
Several fields still have very short field lengths (e.g. the accessibility), and for good reason as they do need to be short. We used to have a character count running as a separate widget. Now that we're on ReactJS, this should be a simple matter to implement a limit display.

**Comment by jkoudys** [comment]
Fixed in 430184ae06fac05bb964487ca95febc913fde657
